requirements.txt

```diff
@@ -1,8 +1,7 @@
 autograd<1.6
 beartype<0.14.0
 dill<0.3.7
-elfi<0.8.7
-fastkde>=1.0.19, <1.0.25
+fastkde>=1.0.24
```

Comment:
This is wrong. Please don't remove the upper bound limit.

Reply:
Whoops, sorry about that, was unintentional. Should be `fastkde>=1.0.24, <1.0.25` correct?

Reply:
yeah. I assume the >=1.0.24 is specific to Python 3.10, right?
it looks like 1.0.24 still supports all the python versions that pnl does so bumping it globally should be ok

Reply:
OK cap is back.